src/med_bench/utils/nuisances.py

```diff
@@ -0,0 +1,511 @@
+from itertools import combinations
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+from numpy.random import default_rng
+from scipy import stats
+from scipy.special import expit
+from scipy.stats import bernoulli
+from sklearn.base import clone
+from sklearn.calibration import CalibratedClassifierCV
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+from sklearn.linear_model import LassoCV, LogisticRegressionCV, RidgeCV
+from sklearn.model_selection import KFold
+from sklearn.preprocessing import PolynomialFeatures
+
+from .utils import _convert_array_to_R, _get_interactions
+
+ALPHAS = np.logspace(-5, 5, 8)
+CV_FOLDS = 5
+TINY = 1.e-12
+
+
+def _get_train_test_lists(crossfit, n, x):
+    """
+    Obtain train and test folds
+
+    Returns
+    -------
+    train_test_list : list, list of index with train and test indexes
+    """
+    if crossfit < 2:
+        train_test_list = [[np.arange(n), np.arange(n)]]
+    else:
+        kf = KFold(n_splits=crossfit)
+        train_test_list = list()
+        for train_index, test_index in kf.split(x):
+            train_test_list.append([train_index, test_index])
+    return train_test_list
+
+
+def _get_regularization_parameters(regularization):
+    """
+    Obtain regularization parameters
+
+    Returns
+    -------
+    cs : each of the values in Cs describes the inverse of regularization strength for predictors
```

Comment:
```suggestion
    cs : list,
           each of the values in cs describes the inverse of regularization strength for predictors
```

Reply:
This has been addressed in commit 14374c26028c5a15b5ce4dd3e8cd907a83d83a37